tests/components/vacuum/test_device_trigger.py

```diff
@@ -118,14 +120,14 @@ def calls(hass):
     hass.states.async_set("vacuum.entity", STATE_CLEANING)
     await hass.async_block_till_done()
     assert len(calls) == 1
-    assert calls[0].data["some"] == "cleaning - device - {} - docked - cleaning".format(
-        "vacuum.entity"
-    )
+    assert calls[0].data[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)